build/code-batch-process.go

```diff
@@ -206,6 +207,21 @@ Example:
 `, "file-batch-exec")
 }
 
+func getGoVersion() string {
+	goModFile, err := os.ReadFile("go.mod")
+	if err != nil {
+		buildInfo, ok := debug.ReadBuildInfo()
```

Comment:
It's incorrect. 

The formatting must use the go.mod version. Otherwise if you have Go 1.20 locally but the CI/CD server has Go 1.19, the formatting result  may differ.

Reply:
Removed fallback